optimize/pom.xml

```diff
@@ -42,17 +42,6 @@
     <url>https://app.camunda.com/jira/browse/OPT/</url>
   </issueManagement>
 
-  <distributionManagement>
```

Comment:
We need to remove the overrides done by Optimize and use the settings inherited from zeebe-parent.

These override can cause problems at the moment of downloading the dependencies from within the runners.